schemas/common/any-wo-array.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$id": "http://jsonlogic.com/schemas/common/any-wo-array.json",
+
+  "title": "All",
+  "description": "Any valid JSON Logic data source.",
```

Comment:
Maybe clarify that this comes without array?

Reply:
Something like `Any valid JSON logic data source other than array.`

Reply:
@mjenny done